pkg/config/parameters.go

```diff
@@ -669,6 +669,9 @@ type Parameters struct {
 	// GlobalExternalAuthorization optionally holds properties of the global external authorization configuration.
 	GlobalExternalAuthorization GlobalExternalAuthorization `yaml:"globalExtAuth,omitempty"`
 
+	// GlobalCircuitBreakerDefaults holds configurable global defaults for the circuit breakers.
+	GlobalCircuitBreakerDefaults *contour_api_v1alpha1.GlobalCircuitBreakerDefaults `yaml:"circuitBreaker,omitempty"`
```

Comment:
Same comments here - let's move under `ClusterParameters`, and pluralize YAML name to `circuitBreakers`.

Reply:
fixed in bdf1de3b2658964a0b571ef62674caaa8efe3ffe